examples/voice-pipeline-agent/fallback_adapter.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+from dotenv import load_dotenv
+from livekit.agents import (
+    AutoSubscribe,
+    JobContext,
+    JobProcess,
+    WorkerOptions,
+    cli,
+    llm,
+    stt,
+    tts,
+)
+from livekit.agents.pipeline import VoicePipelineAgent
+from livekit.plugins import cartesia, deepgram, openai, silero
+
+load_dotenv()
+logger = logging.getLogger("fallback-adapter-example")
+
+
+def prewarm(proc: JobProcess):
+    proc.userdata["vad"] = silero.VAD.load()
+
+
+async def entrypoint(ctx: JobContext):
+    initial_ctx = llm.ChatContext().append(
+        role="system",
+        text=(
+            "You are a voice assistant created by LiveKit. Your interface with users will be voice. "
+            "You should use short and concise responses, and avoiding usage of unpronouncable punctuation."
+        ),
+    )
+
+    await ctx.connect(auto_subscribe=AutoSubscribe.AUDIO_ONLY)
+
+    # wait for the first participant to connect
+    participant = await ctx.wait_for_participant()
+    logger.info(f"starting voice assistant for participant {participant.identity}")
+
+    vad: silero.VAD = ctx.proc.userdata["vad"]
+
+    # fallback to OpenAI if Deepgram goes down
+    fallback_stt = stt.FallbackAdapter(
+        [
+            deepgram.STT(),
+            stt.StreamAdapter(stt=openai.STT(), vad=vad),
+        ]
+    )
+
+    # fallback to OpenAI if Cartesia goes down
+    fallback_tts = tts.FallbackAdapter(
+        [
+            cartesia.TTS(),
+            openai.TTS(),
```

Comment:
nit: use elevenlabs since it supports voice cloning.. OAI doesn't.

```suggestion
            elevenlabs.TTS(),
```